include/deal.II/lac/solver_gmres.h

```diff
@@ -83,6 +83,17 @@ namespace internal
       VectorType &operator() (const unsigned int i,
                               const VectorType   &temp);
 
+      /**
+       * Returns size of data vector. It is used in the solver to store
+       * the Arnoldi vectors.
+       */
+      inline
```

Comment:
Actually, we don't usually use in-class definitions. Please move the function out of line.